addons/mail/static/src/models/activity/activity.js

```diff
@@ -282,7 +295,13 @@ function factory(dependencies) {
          * Do not use this value in a 't-raw' if the activity has been created
          * directly from user input and not from server data as it's not escaped.
          */
-        note: attr(),
+        note: attr({
+            compute: '_computeNote',
+            default: undefined,
+            dependencies: [
+                'note'
```

Comment:
Comma (`,`) after each item (even last one) when using multi-line to define items of array or object.

Reply:
Done